ui/src/app/applications/components/application-resource-tree/application-resource-tree.tsx

```diff
@@ -596,7 +596,7 @@ export const ApplicationResourceTree = (props: ApplicationResourceTreeProps) =>
             if (treeNodeKey(child) === treeNodeKey(root)) {
                 return;
             }
-            if (node.namespace === child.namespace) {
+            if (node.namespace === child.namespace || node.namespace === undefined) {
```

Comment:
Hi @chetan-rns, help me to understand why/when the node namespace will be undefined as opposed to default? This part sets the edges (lines) between 'parent' nodes and the child nodes.  So if the child node's namespace is defined _OR_ undefined, then there will be lines connecting to them.  Is this intentional?